lti-framework/pom.xml

```diff
@@ -15,7 +15,7 @@
 
     <properties>
         <bcprov-jdk15on.version>1.70</bcprov-jdk15on.version>
-        <commons-io.version>2.16.1</commons-io.version>
+        <commons-io.version>2.18.0</commons-io.version>
         <hibernate5-ddl-maven-plugin.version>2.3.0</hibernate5-ddl-maven-plugin.version>
         <reflections.version>0.10.2</reflections.version>
         <spring-security-lti13.version>0.2.0</spring-security-lti13.version>
```

Comment:
There is a 0.3.0 upgrade but it when tried redis complains about not being able to serialize LtiOauth2User